scripts/manage-identity.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+if [ ! -z "${RAD_HOME:-}" ]; then
+    home=$RAD_HOME;
+else
+    home=$HOME;
+fi
+
+if pgrep radicle-proxy; then
+    echo "You need to stop the proxy before switching the identity!"
+    exit 1
+fi
+
+echo "Select the identity you want to switch to or opt to create a new one:"
+
+shopt -s extglob
+select id in $home/.radicle/identities/!(current)/ "NEW IDENTITY"
+do
+case "$id" in
```

Comment:
Is there a reason we don’t use an `if` statement instead of `case`?

Reply:
I think the code looks neater this way and we can easily extend the selection, if we want additional or more specific behavior.